src/token/token.service.ts

```diff
@@ -2,6 +2,7 @@ import { Injectable, Logger } from '@nestjs/common';
 import { Cron } from '@nestjs/schedule';
 import { InjectRepository } from '@nestjs/typeorm';
 import axios from 'axios';
+import { configService } from 'src/config.service';
```

Comment:
Please use relative paths to avoid build errors that can manifest in some cases.